modules/tensor_mechanics/src/userobjects/CavityPressureUserObject.C

```diff
@@ -67,17 +67,17 @@ CavityPressureUserObject::CavityPressureUserObject(const InputParameters & param
 }
 
 Real
-CavityPressureUserObject::getValue(const std::string & quantity) const
+CavityPressureUserObject::getValue(const MooseEnum & quantity) const
 {
   Real value = 0;
-  if ("initial_moles" == quantity)
+  if (quantity == "initial_moles")
```

Comment:
@bwspenc helped me with the enums recently: I believe if you make a namespace for the enum, the comparison with the enum is faster than using a string comparison, but @bwspenc will know better than me

Reply:
I made this switch so that there would be an enum to list the options in the documentation.  If you point me to an example, I'll also do the namespace business.